### ISSUE TITLE
md-input: tabbing to filled input doesn't read the label

### ISSUE THREAD

**Issue by iveysaur** [angular/material2#605]
Moving to a filled input using tab doesn't read the label.

Steps:
1. Use tab to move to an input that has been filled

The label for that input will not be read and instead it will announce that the text has been selected.

Found using the screenreader on OSX El Capitan in Chrome

**Comment by mmalerba** [comment]
seems to be working for input-container on the dev app
